include/dem/dem.h

```diff
@@ -364,6 +375,12 @@ class DEMSolver
   PVDHandler                           particles_pvdhandler;
   const unsigned int                   standard_deviation_multiplier;
 
+  std::unordered_map<int, Tensor<1, dim>> momentum;
+  std::unordered_map<int, Tensor<1, dim>> force;
+  std::unordered_map<int, double>         displacement;
+  std::unordered_map<int, double>         MOI;
+  std::unordered_map<int, Tensor<1, dim>> acceleration;
```

Comment:
Make the key of these map to be ParticleIndex:
https://www.dealii.org/current/doxygen/deal.II/namespacetypes.html#ae4ea18e3efb31f0312a9e754873d71b8
This way, it will be more coherent. Additionally, particle indices are unsigned int and not int, this should prevent some weird casting that might occur and make access potentially slightly faster.

Reply:
changed to ParticleIndex